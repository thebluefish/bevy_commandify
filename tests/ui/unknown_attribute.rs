use bevy_commandify::*;

#[command(fail)]
fn foo(world: &mut World) { }

fn main() { }