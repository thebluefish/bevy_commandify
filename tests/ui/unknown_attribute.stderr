error: Unknown attribute `fail`
 --> tests/ui/unknown_attribute.rs:3:11
  |
3 | #[command(fail)]
  |           ^^^^
